pymc_marketing/mmm/base.py

```diff
@@ -88,6 +88,14 @@ def preprocessing_methods(self) -> List[Callable[[pd.DataFrame], pd.DataFrame]]:
             if getattr(method, "_tags", {}).get("preprocessing", False)
         ]
 
+    def get_target_transformer(self):
+        try:
+            return self.target_transformer
+        except AttributeError:
+            null_scaler = MaxAbsScaler()
+            null_scaler.fit([[0], [1]])
```

Comment:
Is this to create a "trivial" scaler?

Reply:
Yes. I wanted a scaler that did nothing. Is there a better way?

Reply:
I think this is ok! I wonder if is an overkill just adding a `IdentityTransformer` just for the sake of readability...

Reply:
I don't see any `IdentityTransformer` in the sklearn documentation. From what I saw, it looks like a [`FunctionTransformer`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.FunctionTransformer.html#sklearn.preprocessing.FunctionTransformer) with `func=None` would do the job.

Reply:
As you wish :) No strong opinions here.